javaagent-tooling/javaagent-tooling.gradle

```diff
@@ -28,7 +28,12 @@ dependencies {
   implementation deps.opentelemetrySdk
   implementation deps.opentelemetrySdkAutoconfigure
   implementation deps.opentelemetrySdkMetrics
-  implementation deps.opentelemetryKotlin
+  implementation (deps.opentelemetryKotlin) {
```

Comment:
@anuraaga do you think that Kotlin extension should bring Kotlin with it? It should have `compileOnly` dependency on Kotlin stdlib?